src/test/java/codeu/controller/LoginServletTest.java

```diff
@@ -52,20 +52,10 @@ public void testDoGet() throws IOException, ServletException {
     Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
   }
 
-  @Test
-  public void testDoPost_BadUsername() throws IOException, ServletException {
-    Mockito.when(mockRequest.getParameter("username")).thenReturn("bad !@#$% username");
-
-    loginServlet.doPost(mockRequest, mockResponse);
-
-    Mockito.verify(mockRequest)
-        .setAttribute("error", "Please enter only letters, numbers, and spaces.");
-    Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
-  }
-
   @Test
   public void testDoPost_NewUser() throws IOException, ServletException {
```

Comment:
Rename test case.  This is now testing for attempt logins for users that aren't registered.